R/pkg/R/DataFrame.R

```diff
@@ -293,6 +294,121 @@ setMethod("names<-",
             }
           })
 
+#' @rdname columns
+#' @name colnames
+setMethod("colnames",
+          signature(x = "DataFrame"),
+          function(x, do.NULL = TRUE, prefix = "col") {
```

Comment:
Is it possible to remove default value for do.NULL and prefix, and throws unsupported message when they are not missing?